recipes/silicon/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86]
```

Comment:
Please remove this. 10.13 is the new global pin

Reply:
ahh right, do i have to re-render instead? Didn't build without this, maybe because the PR is quite old

Reply:
There is no rerender on staged-recipes. You probably need to merge/rebase on main.